runner.go

```diff
@@ -854,7 +859,7 @@ func atomicWrite(path string, contents []byte, perms os.FileMode, backup bool) e
 		}
 	}
 
-	if err := os.Rename(f.Name(), path); err != nil {
```

Comment:
This seems like it may introduce some non-atomicity when swapping the file since this does an open, and then an io copy from Go, whereas a rename is usually an atomic thing. If the service was watching for changes to this file it seems like it may see some intermediate state. I'm not sure how we'd fix this in a general way unless consul-template gets a scratch dir to work in under the parent, but that's ugly.